sql/src/main/java/io/crate/analyze/where/WhereClauseAnalyzer.java

```diff
@@ -238,9 +234,12 @@ private WhereClause tieBreakPartitionQueries(EvaluatingNormalizer normalizer,
         }
         if (canMatch.size() == 1) {
             Tuple<Symbol, List<Literal>> symbolListTuple = canMatch.get(0);
-            WhereClause whereClause = new WhereClause(symbolListTuple.v1());
-            whereClause.partitions(symbolListTuple.v2());
-            return whereClause;
+            WhereClause where = new WhereClause(symbolListTuple.v1(),
+                    whereClause.primaryKeys().orNull(),
+                    new ArrayList<String>(symbolListTuple.v2().size()),
```

Comment:
jep, we have to copy the partitions and ids too since we now have all information on the whereclause. there are a few other places where we have to partially copy the whereclause.